website/docs/compute/get-started.md

```diff
@@ -0,0 +1,205 @@
+# Getting started with Compute
+
+## Installation
+
+To use Compute, you'll need an environment token and the Subo command line 
+tool. The environment token allows Compute to authenticate with 
+our service that manages metadata storage, telemetry, and more.
+
+### Install Subo: the Compute CLI
+
+If you use macOS, you can use [Homebrew](https://brew.sh) to install the 
+`subo` command line tool:
+
+```bash
+brew tap suborbital/subo
+brew install subo
+```
+
+Note: this may take a few minutes! Next, run `subo --version` to ensure the 
+installation was successful. 
+
+To install on Linux (or macOS without Homebrew), you can [download Subo directly](https://github.com/suborbital/subo/releases).
+
+## Generate your token
+You can get a Compute Environment token either with our [environment token generator web app](https://suborbital.network/) or with the `subo` command line tool.
+
+:::caution  
+The domain of your email address should match the `environment` you 
+configure for Compute; i.e., if your email is `laika@example.com`, your 
+Compute environment would be called `com.example`. See [Fully-qualified 
+function names](./customizing-functions/fully-qualified-function-names.md) 
+for more 
+information.
+:::
+
+To create your token, run:
+
+```bash
+subo compute create token <email>
+```
+
+A verification code will be sent to your email address, and the token will be 
+used to authenticate you and link your Compute installation to you.
+
+:::tip
+Subo will print out your token in your terminal, and cache it for use in 
+subsequent steps.
+:::
+
+Keep your environment token safe (store it in your password manager), and 
+only use it when running or installing Compute. If you lose it, just repeat 
+this token generation process.
+
+
+## Run Compute locally
+
+To run Compute locally, you'll use the Subo CLI and [Docker](https://docs.docker.com), both of which must be installed before continuing.
+
+### Prepare your local environment
+
+The Subo CLI will create some files on disk to set up your local deployment, so get started by creating a `suborbital` directory somewhere on your filesystem:
+
+```bash
+mkdir suborbital
+cd suborbital
+```
+
+### Start Compute
+
+Next, use Subo to start your local Compute Core instance:
+
+```bash
+subo compute deploy core --local
```

Comment:
Note that on some systems this will fail ( suborbital/subo#345 ) before we even told people about the `--proxy-port` option.

Reply:
I'll create an issue to document this after this one merges.

Reply:
Here's the issue! https://github.com/suborbital/docs/issues/160